src/services/feed_validator.py

```
@@ -2,7 +2,7 @@
 import asyncio
 import feedparser
 import logging
-from typing import Tuple, Optional, Dict
+from typing import Tuple, Optional
```

Ruff (0.8.0)

`typing.Tuple` is deprecated, use `tuple` instead

(UP035)